curvefs/src/client/metric/client_metric.h

```diff
@@ -289,13 +289,41 @@ struct DiskCacheMetric {
           diskUsedBytes(prefix, fsName + "_diskcache_usedbytes", 0) {}
 };
 
-struct KVClientMetric {
+struct KVClientManagerMetric {
     static const std::string prefix;
-    InterfaceMetric kvClientGet;
-    InterfaceMetric kvClientSet;
 
-    KVClientMetric()
-        : kvClientGet(prefix, "get"), kvClientSet(prefix, "set") {}
+    std::string fsName;
+    InterfaceMetric get;
+    InterfaceMetric set;
+    // kvcache count
+    bvar::Adder<uint64_t> count;
+    // kvcache hit
+    bvar::Adder<uint64_t> hit;
+    // kvcache miss
+    bvar::Adder<uint64_t> miss;
+
+    explicit KVClientManagerMetric(const std::string& name = "")
+        : fsName(!name.empty() ? name
+                               : prefix + curve::common::ToHexString(this)),
+          get(prefix, fsName + "_get"),
+          set(prefix, fsName + "_set"),
+          count(prefix, fsName + "_count"),
+          hit(prefix, fsName + "_hit"),
+          miss(prefix, fsName + "_miss") {}
+};
+
+struct MemcacheClientMetric {
+    static const std::string prefix;
+
+    std::string fsName;
+    InterfaceMetric get;
+    InterfaceMetric set;
```

Comment:
what's the difference between `KVClientManagerMetric`'s get/set ?

Reply:
> what's the difference between `KVClientManagerMetric`'s get/set ?

manager will put set task to queue.